explain-1010/ContentView.swift

```diff
@@ -13,7 +13,7 @@ struct ContentView: View {
             Image(systemName: "globe")
                 .imageScale(.large)
                 .foregroundColor(.accentColor)
-            Text("테스트 테스트 테스트")
+            Text("어디신디")
         }
         .padding()
```

Comment:
```suggestion
            Text("좀 아쉬운데요?")
        }
        .padding()
```

~~ 이렇게 하면 더 좋을 거 같습니다.